pom.xml

```diff
@@ -1646,6 +1674,18 @@
 <!-- Duplicate entry for javax.websocket-javax.websocket-api-1.1 skipping -->
 <!-- WebSocketConnector end -->
 
+  <dependency>
+      <groupId>au.edu.federation.caliko</groupId>
+      <artifactId>caliko</artifactId>
+      <version>1.3.8</version>
+    </dependency> 
+    
+        <dependency>
+      <groupId>au.edu.federation.caliko.visualisation</groupId>
```

Comment:
the pom for this needs to be added to the mrl repo for it to build and cache the jar properly.

Reply:
hmm .. thought i put it there ..
https://repo.myrobotlab.org/artifactory/myrobotlab/au/edu/federation/caliko/visualisation/caliko-visualisation/1.3.8/

Reply:
mebbe i messed up the path